watchtower-plugin/src/net/http.rs

```diff
@@ -154,6 +155,24 @@ pub async fn send_appointment(
     }
 }
 
+/// Handles a ping request to the tower.
+pub async fn ping(server_url: &str) -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
I think we should be able to replace `Box<dyn std::error::Error>` with just a `String`.

Reply:
done!!